web/react/components/signup_user_complete.jsx

```diff
@@ -117,24 +117,23 @@ module.exports = React.createClass({
 
         var signup_message;
         if (auth_services.indexOf("gitlab") >= 0) {
-            signup_message = <p>{"Choose your username and password for the " + this.props.teamDisplayName + " " + strings.Team} <a href={"/"+this.props.teamName+"/signup/gitlab"+window.location.search}>{"or sign up with GitLab."}</a></p>;
-        } else {
-            signup_message = <p>{"Choose your username and password for the " + this.props.teamDisplayName + " " + strings.Team + "."}</p>;
+            signup_message = <div className="signup-custom__container"><a className="btn btn-custom-login gitlab" href={"/"+this.props.teamName+"/signup/gitlab"+window.location.search}><span className="icon" />{"with GitLab"}</a>
+            <div className="or__container"><span>or</span></div></div>;
         }
 
         return (
             <div>
                 <img className="signup-team-logo" src="/static/images/logo.png" />
-                <h4>Welcome to { config.SiteName }</h4>
+                <h3 className="text-center">Signup to { config.SiteName }</h3>
                 <div className="form-group form-group--small">
                     <span></span>
                 </div>
                 { signup_message }
-                <p>Your username can be made of lowercase letters and numbers.</p>
```

Comment:
I noticed that this is repeated in the oauth user signup page but if it is removed here will users who are completing the normal sign-up flow see this anywhere else?  I just remember this previously being an issue with signup flow due to users not realizing that they couldn't use uppercase letters until an error message appeared which isn't good.